### ISSUE TITLE
No output in the console logs when no accessibility issues found

### ISSUE THREAD

**Issue by cipivanov** [abhinaba-ghosh/axe-playwright#21]
It would be nice to print a small message to the console mentioning the lack of accessibility issues explicitly rather than relying on indirect no output behaviour for that case.

**Comment by abhinaba-ghosh** [comment]
Available in `v1.1.7`

**Comment by mririgoyen** [comment]
How do I disable these messages? I only care if things fail and the extra 4-5 lines of console span per passing test is messy.

**Comment by StephanBijzitter** [comment]
Uhhh I'm not sure I understand, I'm happy with specifying includedImpacts to tailor the output to my needs
